### PR TITLE
fix(ActivityCenter): Fix positioning AC popup

### LINKED ISSUE #8030: [ActivityCenter] should open always below the bell icon
### Description
Currently if the members list is opened, the bell icon is move to the left, however activity center opens always at the very right side. It should open below the bell icon like shown in image

Ac

### LINKED ISSUE #8124: change tooltip from 'Activity' to 'Notifications'
### Description

Screen Shot 2022-11-04 at 10 28 49 AM

### PR DESCRIPTION
**Pull request by MishkaRogachev**
Close #8030, #8124

Fix positioning AC popup & tooltip text

### Affected areas

Activity Center, all screens

### Screenshot of functionality (including design for comparison)

Screenshot 2022-11-04 at 17 25 44

### CODE REVIEW

File: ui/app/mainui/AppMain.qml [hunk 1]
```diff
@@ -990,8 +990,12 @@ Item {
             id: activityCenterPopupComponent
             ActivityCenterPopup {
                 id: activityCenter
-                height: appView.height - 56 * 2 // TODO get screen size // Taken from old code top bar height was fixed there to 56
-                y: 56
+                // TODO get screen size // Taken from old code top bar height was fixed there to 56
+                property int _buttonSize: 56
```
Comment (caybro) [inline ui/app/mainui/AppMain.qml:994]: How do you know it will be 56?

Comment (MishkaRogachev) [reply ui/app/mainui/AppMain.qml:994]: There are todo for what. We have no such constant in style/theme currently

**Comment by elina2015** [comment]
@MishkaRogachev hey, could you rebuild the pr, so here will be a build for macOS too? Thank you in advance!

**Review by elina2015 — APPROVED** [review]
Tested and approved

File: ui/app/mainui/AppMain.qml (hunk 1 above)
Comment (PascalPrecht) [reply ui/app/mainui/AppMain.qml:994]: @caybro any other suggestions here? Cause otherwise we can go with this for now.

**Review by borismelnik — APPROVED** [review]
👍

File: ui/app/mainui/AppMain.qml (hunk 1 above)
Comment (caybro) [reply ui/app/mainui/AppMain.qml:996]: Nope

**Review by caybro — APPROVED** [review]

**Pull request merged**